### ISSUE TITLE
Drawing "undesired" shapes in mouse mode

### ISSUE THREAD

**Issue by mauriva** [mohammadKarimi/Riter#149]
I am following the development of this tool with interest and have briefly tested various versions, including the latest 0.2.4 on Windows 11 24H2. In all the versions I tested, I noticed what I consider to be a bug. Here’s what I do:  

1) I start Riter and draw something on the screen (freehand, lines, rectangles, ...)  
2) I select the command to use the mouse normally (first button on the left)
3) In "mouse" mode, by clicking (multiple times) on the drawn shapes, almost always Riter starts drawing shapes again (especially rectangles!) even though the corresponding button has not been selected.  

I don't understand if this happens only to me or if it is indeed a bug in the program.

**Comment by mohammadKarimi** [comment]
Thank you for your issue, I will check it again as you did. to see the bug, if it is still there, I will fix it.

**Comment by mauriva** [comment]
I report that the bug is still present, even in the latest 0.2.6.
I reproduced it exactly as described in the first post.
I've noticed that in ‘mouse mode’ when you place the cursor over one of the drawn shapes or a line drawn, at some point it becomes a small dot and starts drawing again from there, even though mouse mode is still active.
I have noticed that this does not always occur, but I am still not sure when this happens...
